helm/lib/dependabot/helm/update_checker.rb

```diff
@@ -217,9 +217,8 @@ def extract_repo_name(repo_url)
       sig { params(index_url: String).returns(T.nilable(T::Hash[T.untyped, T.untyped])) }
       def fetch_helm_chart_index(index_url)
         Dependabot.logger.info("Fetching Helm chart index from #{index_url}")
-
         response = Excon.get(
-          index_url,
+          index_url.gsub("oci://", "https://"),
```

Comment:
this feels implicit, should we have an explicit step that doesn't happen inside the Excon.get call?